src/SkeletalTrapezoidation.h

```diff
@@ -577,7 +577,7 @@ class SkeletalTrapezoidation
     /*!
      * add a new toolpath segment, defined between two extrusion-juntions
      */
-    void addToolpathSegment(const ExtrusionJunction& from, const ExtrusionJunction& to, bool is_odd, bool force_new_path);
+    void addToolpathSegment(const ExtrusionJunction& from, const ExtrusionJunction& to, bool is_odd, bool force_new_path, bool from_is_3way, bool to_is_3way);
```

Comment:
The intended use for `from_is_3way` and `to_is_3way` is not clear to me at first glace. Maybe at them to the `\param` comments?

Reply:
Done